bark/world/world.hpp

```diff
@@ -128,6 +128,8 @@ class World : public commons::BaseType {
 
   bool GetRemoveAgents() { return remove_agents_; }
 
+  float GetFracLateralOffset() const { return frac_lateral_offset_; }
```

Comment:
in general should be not use double as a few functions below we use that. I do not like the mixing of types